docs/roles/integrator/integrating.md

```diff
@@ -0,0 +1,372 @@
+http post http://rpc.nearprotocol.com jsonrpc=2.0 method=query  \
+                                      params:="[\"account/test.near\",\"\"]" \
+                                      id="placeholder"
+# the request above returns view of account information
+# ------------------------------------------------------------------------------
+# response
+{
+  "id": "placeholder",
+  "jsonrpc": "2.0",
+  "result": {
+    "amount": "1000000000000000011",
+    "code_hash": "11111111111111111111111111111111",
+    "locked": "0",
+    "storage_paid_at": 0,
+    "storage_usage": 182
+  }
+}
+
+# this example used HTTPie - a CLI, cURL-like tool for humans available at http://httpie.org
+```
+
+#### Resources
+
+- RPC interface in related documentation [here](https://docs.nearprotocol.com/docs/interaction/rpc)
+- Consensus and finality here [NEAR Lunch and Learn Ep. 04: Nightshade: Consensus and finality](https://www.youtube.com/watch?v=k2ziZiZWquQ&list=PL9tzQn_TEuFW_t9QDzlQJZpEQnhcZte2y&index=4)
+
+### `/status` Endpoint
+
+The `/status` endpoint is available for querying the most current state of the blockchain.
+
+For example, TestNet status is available via `http://rpc.nearprotocol.com/status`
+
+An `HTTP GET` request to this endpoint produces the following output:
+
+```js
+{
+  "chain_id": "testnet", // universal name of the chain, configured in ~/.near/genesis.json
+  "rpc_addr": "0.0.0.0:8080",
+  "sync_info": {
+    "latest_block_hash": "9MZPsj3fzF2tQtPsZz12M1XKCWZHcZU1QFjBw9coHboW",
+    "latest_block_height": 12793,
+    "latest_block_time": "2019-12-17T07:01:09.474230160Z",
+    "latest_state_root": "FFrjV9UrH5gT4mwUYuat6xr19vVATWJstatnUj9V196z",
+    "syncing": false
+  },
+  "validators": [
+    {
+      "account_id": "far",
+      "is_slashed": false
+    }
+  ],
+  "version": {
+    "build": "c4046b02-modified",
+    "version": "0.4.10"
+  }
+}
+```
+
+## Accounts
+
+### Accounts
+
+The NEAR platform provides an account name system with a scoping structure similar to the domain name system.  You can read more about [accounts on the NEAR platform here](https://nomicon.io/Primitives/Account.html).
+
+Each account lives on one and only one shard.
+
+All related data for an account also lives on the same shard including:
+- Account Balance
+- Locked balance (for staking)
+- Code of the contract
+- Key-value storage of the contract. Stored in a ordered trie
+- Access Keys
+- Postponed ActionReceipts
+- Received DataReceipts
+
+### Access Keys
+
+Each account can have an unlimited number of access keys.
+
+An access key grants access to the resources of an account on the system. Each access key is identified by a unique public key. This public key is used to validate signatures on transactions. Each access key contains a unique `nonce` to differentiate and sort transactions signed with the access key.
+
+Access keys represent a specific level of authorization.  Currently the NEAR platform supports two types of access keys:
```

Comment:
`Access keys represent a specific level of access to the account and operating under it's name with other accounts/contracts`

Reply:
more clear, thank you. will update

Reply:
✅